schema/airport.graphql

```diff
@@ -0,0 +1,12 @@
+type GetAirport {
+    fuzzySearch(query: String!): [Airport]
+    fromICAO(code: String!): Airport
```

Comment:
Can they be called instead just `search` and `byICAO`?

Reply:
sure - but any particular reason?

Reply:
- KISS: `fuzzy` implies there can be a non-fuzzy search, though that's not in-sight to be developed
- all getters or async functions are usually named starting with `by…`, or at least that's what I've perceived being more common

Reply:
the search one makes sense, sure.

but for the ICAO, `byICAO` would imply "all of them, sorted by ICAO" or "all of them in a dict with ICAO as the key", so I think 'from' makes more sense here (it's more like "give me airport details based on the ICAO I give you").